lib/src/client/call.dart

```diff
@@ -112,6 +119,9 @@ class ClientCall<Q, R> implements Response {
   }
 
   void _terminateWithError(GrpcError error) {
+    _timeline?.finish(arguments: {
```

Comment:
I'm not seeing where `_timeline` finishes without error. Where does this happen?

Reply:
Good point! I forgot to finish at normal exit.

Reply:
looks good!